pep-0215.txt

```diff
@@ -69,7 +69,7 @@ Examples
 ========
 
 Here is an example of an interactive session exhibiting the
-expected behaviour of this feature::
+expected behaviour of this feature. ::
```

Comment:
No, since there is a space between `.` and `::`, colon not rendered.
But I prefer put `::` in own line, as @brettcannon's comment.